### ISSUE TITLE
[`Pylint`] `PLE0241` fix can delete comments

### ISSUE THREAD

**Issue by MeGaGiGaGon** [astral-sh/ruff#18814]
### Summary

The fix for [duplicate-bases (PLE0241)](https://docs.astral.sh/ruff/rules/duplicate-bases/#duplicate-bases-ple0241) can delete comments. In this case it should either be marked unsafe, not apply, or move the comments. [playground](https://play.ruff.rs/5bb7a327-e8eb-4cc0-872e-35eb2a616a29)
```
PS D:\python_projects> Get-Content issue.py
```
```py
class Bar(Foo,  # 1
    Foo  # 2
):
    pass
```
```
PS D:\python_projects> uvx ruff check issue.py --isolated --select PLE
```
```snap
issue.py:2:5: PLE0241 [*] Duplicate base `Foo` for class `Bar`
  |
1 | class Bar(Foo,  # 1
2 |     Foo  # 2
  |     ^^^ PLE0241
3 | ):
4 |     pass
  |
  = help: Remove duplicate base

Found 1 error.
[*] 1 fixable with the `--fix` option.
```
```
PS D:\python_projects> uvx ruff check issue.py --isolated --select PLE --fix --diff
```
```diff
--- issue.py
+++ issue.py
@@ -1,4 +1,3 @@
-class Bar(Foo,  # 1
-    Foo  # 2
+class Bar(Foo  # 2
 ):
     pass

Would fix 1 error.
```

### Version

ruff 0.12.0 (87f0feb 2025-06-17) + playground

**Comment by MichaReiser** [comment]
The fix should be marked as unsafe if it intersects with any comments and the fix safety should be documented.
